x-pack/solutions/security/plugins/security_solution/public/common/hooks/use_url_state.ts

```diff
@@ -16,11 +16,11 @@ import { useQueryTimelineByIdOnUrlChange } from './timeline/use_query_timeline_b
 export const useUrlState = () => {
   useSyncGlobalQueryString();
   useInitSearchBarFromUrlParams();
-  useInitTimerangeFromUrlParam();
-  useUpdateTimerangeOnPageChange();
   useInitTimelineFromUrlParam();
   useSyncTimelineUrlParam();
   useQueryTimelineByIdOnUrlChange();
+  useInitTimerangeFromUrlParam();
+  useUpdateTimerangeOnPageChange();
```

Comment:
If we let `useInitTimelineFromUrlParam` run after `useInitTimerangeFromUrlParam` , the timerange is overwritten by `useInitTimelineFromUrlParam`. This makes sure global such as `timerange` are not overwritten from any other hook.